### ISSUE TITLE
I'd like to contribute a save button to inspector

### ISSUE THREAD

**Issue by naugtur** [aframevr/aframe-inspector#517]
Hi,

the "download HTML" button is a great functionality if I want to teach A-Frame to people with no programming background whatsoever. 
I'd like to find a simple way to let them use inspector online and save the HTML back to the jsfiddle (or anything similar) they're using. I could also look into building something on Glitch instead.

I'm asking you @maintainers to point me the right way so I know where to start (to programatically get the HTML) and let me know if anyone's already working on a similar functionality.

**Comment by naugtur** [comment]
@ngokevin I'm teaching 12 year olds some Aframe this Saturday and I thought I'd revisit this idea. Do you have any pointers for me  where to start?

I'm going to use the clipboard functionality for now, but I'm having trouble with that. It's adding a canvas to the result and then when I reopen inspector on the new sources it won't work when the canvas is there.

**Comment by ngokevin** [comment]
I've removed the export to HTML feature. Inspector was not made well for creating projects from scratch without source code, and export to HTML could never work without source code. There are other and better projects for creating scenes entirely visually, including those made with A-Frame (hologram.cool, Ottifox).

I made https://github.com/supermedium/aframe-watcher to save back to file though, and can still copy back individual entities / values.
